docs/wiki/Troubleshooting-Problems.md

```diff
@@ -0,0 +1,128 @@
+| Native DCR wallet | ``mainnet/assetdb/dcr/mainnet/spvlogs/neutrino.log`` |
+| Native BTC wallet | ``mainnet/assetdb/btc/mainnet/logs/neutrino.log``    |
+| Native LTC wallet | ``mainnet/assetdb/ltc/mainnet/logs/neutrino.log``    |
+
+# Native Wallets
+
+## Rescanning
+
+_**NOTE:** This section of the is only applicable to Decred, Bitcoin and Litecoin wallets._
+
+In rare circumstances, your wallet may need to rescan the blockchain for missed transactions. 
+This is a process that can take several hours, depending on the number of transactions in the wallet.
+To rescan the wallet, open the [wallet settings](Wallet#wallet-settings) dialog and click the `Rescan` button. 
+
+<img src="./images/using-bison-wallet/wallets-settings-rescan.png" width="400" alt="">
+
+You will a message with a green check icon if the rescan was successfully initiated.
+
+<img src="./images/using-bison-wallet/wallets-settings-rescan-running.png" width="400" alt="">
+
+## Recovery
+
+_**NOTE:** This section of the is only applicable to Neutrino based wallets like Bitcoin and Litecoin._
+
+If a neutrino wallet is corrupted, the [dexc.log](#location-of-application-and-log-files) may have a message
+similiar to:
+
+``
+[ERR] CORE: Unable to connect to btc wallet (start and sync wallets BEFORE starting dex!): 
+connectWallet: failed to connect btc wallet: connect failure: wallet not found
+``
+
+Note `btc` in the above message, and `wallet not found`. 
+This may also say something about db corruption, or similar errors, which may also be addressed by
+recovering the wallet.
+
+In this case, the [Wallet](wallet) will still show the BTC wallet since it was previously 
+configured, but it will not be synchronized and cannot be unlocked or connected.
+
+<img src="./images/using-bison-wallet/wallets-settings-recover-wallet.png" width="550" alt="">
+
+To recover the wallet, open the [wallet settings](Wallet#wallet-settings) for the affected wallet, and click the 
+`Recover` button:
+
+<img src="./images/using-bison-wallet/wallets-settings-recover.png" width="400" alt="">
+
+A warning will appear indicating what actions to expect, click ``Submit``.
+
+<img src="./images/using-bison-wallet/wallets-settings-recover-warning.png" width="400" alt="">
+
+Once the recovery process has completed, the wallet will be re-created and start
+synchronizing:
+
+<img src="./images/using-bison-wallet/wallets-settings-recovered.png" width="550" alt="">
+
+## Forced Rescan
+
+
+>[!CAUTION]
+>The recommended procedure for rescanning is through the [Wallet Settings](Wallet#wallet-settings)
+as described in the [Rescanning](#rescanning) section of this wiki.
+
+If you wish to force a rescan, you can do so by following these steps:
+
+1. Delete the wallet.db file in the `assetdb/btc/mainnet/` (for mainnet) folder as described at the end 
+of the previous section.  Start DEX and log in. Follow the instructions in 
+[recovery section](#recovery) to reinitialize the wallet and begin a resync.
```

Comment:
imo its always better to move files rather than delete, sort of like what Windows does with the recycling bin. I think it's better to advise a user to move the wallet.db here as well, rather than delete.

Reply:
Would renaming to wallet.db.bak do it? seems easier than saying to move.

Reply:
nm, went with "move to a safe backup location" since we need to mvoe the complete folder in the next section anyway.